packages/manager/src/features/Databases/DatabaseLanding/DatabaseActionMenu.tsx

```diff
@@ -3,36 +3,69 @@ import { useHistory } from 'react-router-dom';
 
 import { ActionMenu } from 'src/components/ActionMenu/ActionMenu';
 
-import type { Engine } from '@linode/api-v4';
+import type { DatabaseStatus, Engine } from '@linode/api-v4';
 import type { Action } from 'src/components/ActionMenu/ActionMenu';
+import { useIsDatabasesEnabled } from '../utilities';
+import { useResumeDatabaseMutation } from 'src/queries/databases/databases';
+import { enqueueSnackbar } from 'notistack';
+import { getAPIErrorOrDefault } from 'src/utilities/errorUtils';
+import { APIError } from '@linode/api-v4/src/types';
 
 interface Props {
   databaseEngine: Engine;
   databaseId: number;
   databaseLabel: string;
   handlers: ActionHandlers;
+  databaseStatus: DatabaseStatus;
 }
 
 export interface ActionHandlers {
   handleDelete: () => void;
   handleManageAccessControls: () => void;
   handleResetPassword: () => void;
+  handleSuspend: () => void;
 }
 
 export const DatabaseActionMenu = (props: Props) => {
-  const { databaseEngine, databaseId, databaseLabel, handlers } = props;
+  const {
+    databaseEngine,
+    databaseId,
+    databaseLabel,
+    handlers,
+    databaseStatus,
+  } = props;
 
-  const databaseStatus = 'running';
-  const isDatabaseNotRunning = databaseStatus !== 'running';
+  const { isDatabasesV2GA } = useIsDatabasesEnabled();
+  const { mutateAsync: resumeDatabase } = useResumeDatabaseMutation(
+    databaseEngine,
+    databaseId
+  );
+
+  const status = 'running';
+  const isDatabaseNotRunning = status !== 'running';
+  const isDatabaseSuspended =
+    databaseStatus === 'suspended' || databaseStatus === 'suspending';
 
   const history = useHistory();
 
+  const handleResume = () => {
+    resumeDatabase()
+      .then(() => {
+        enqueueSnackbar('Database Cluster resumed successfully.', {
+          variant: 'success',
+        });
+      })
+      .catch((e: APIError[]) => {
+        const defaultError =
+          'There was an error resuming this Database Cluster.';
+        const error = getAPIErrorOrDefault(e, defaultError)[0].reason;
+        enqueueSnackbar(error, {
+          variant: 'error',
+        });
+      });
+  };
+
```

Comment:
nitpicky: Can we use async/await to improve readability ?

```suggestion
const DEFAULT_RESUME_ERROR = 'There was an error resuming this Database Cluster.';

const handleResume = async () => {
  try {
    await resumeDatabase();
    return enqueueSnackbar('Database Cluster resumed successfully.', { variant: 'success' });
  } catch (e: APIError[]) {
    const error = getAPIErrorOrDefault(e, DEFAULT_RESUME_ERROR)[0].reason;
    return enqueueSnackbar(error, { variant: 'error' });
  }
};
```

Reply:
I made this change in branch that we merged in. (Sam is on vacation.)